tests/yam/agama/agama.pm

```diff
@@ -18,14 +20,25 @@ use testapi qw(
 sub run {
     my $self = shift;
     my $test = get_required_var('AGAMA_TEST');
+    my $configure_dasd = get_var('CONFIGURE_DASD');
     my $reboot_page = $testapi::distri->get_reboot_page();
 
+    # For testing purposes remove before merge
+    if (is_s390x() && is_backend_s390x() && get_var('AGAMA_DELAY')) {
+      print("Sleeping 30 minutes for testing purposes");
+      sleep(1800);
+    }
+
     script_run("dmesg --console-off");
-    assert_script_run("/usr/share/agama/system-tests/" . $test . ".cjs", timeout => 1200);
+    assert_script_run("CONFIGURE_DASD=${configure_dasd} /usr/share/agama/system-tests/" . $test . ".cjs", timeout => 1200);
     script_run("dmesg --console-on");
 
-    select_console('installation');
-    $reboot_page->reboot();
+    if (is_s390x() && is_backend_s390x()) {
```

Comment:
I guess it will suffice to use `is_s390x`.

Reply:
Done 4740dff6fd12210421ee398a025c7823e0e963f7